@navikt/core/react/src/form/combobox/FilteredOptions/FilteredOptions.tsx

```diff
@@ -23,10 +23,10 @@ const FilteredOptions = () => {
     isMouseLastUsedInputDevice,
     isValueNew,
   } = useFilteredOptionsContext();
-  const { maxSelected } = useSelectedOptionsContext();
+  const { maxSelected, isMultiSelect } = useSelectedOptionsContext();
 
   const shouldRenderNonSelectables =
-    maxSelected.isLimitReached || // Render maxSelected message
+    isMultiSelect || // Render maxSelected message
```

Comment:
If i'm not missing anything, you can have a multiselect Combobox without `maxSelected` set. So need to check if maxSelected is defined here also.